storage/PreloaderThread.cpp

```diff
@@ -79,7 +86,8 @@ std::size_t PreloaderThread::preloadNUMAAware(
     const std::size_t num_slots) {
 #ifdef QUICKSTEP_HAVE_LIBNUMA
   std::size_t blocks_loaded = 0;
-  const NUMAPlacementScheme *placement_scheme = relation.getNUMAPlacementSchemePtr();
+  const NUMAPlacementScheme *placement_scheme =
+      relation.getNUMAPlacementSchemePtr();
```

Comment:
This is a late comment replied to #206.

I would suggest to define the method `preloadNUMAAware` under `QUICKSTEP_HAVE_LIBNUMA`.

To me, it does not make sense to build this method on a machine without `libnuma`.